### DISCUSSION TITLE
Field to insert your own instance name, if it's filled the links leads to my instance

### DISCUSSION THREAD

**Discussion by Ciantic** [tvler/streetpass#2] [Ideas]
Idea is to make following easier. Allow user to fill their own instance name, and make links to work with that.

Example:

My instance is https://twit.social

Suppose I visit https://daringfireball.net

<img src="https://user-images.githubusercontent.com/64731/217041349-68ed89db-0e05-4713-aed9-9e16e78623c5.png" width=300 />

If I click that "mastodon.social/@gruber" it could link to https://twit.social/@gruber@mastodon.social

**Comment by tvler** [comment]
This has been shipped! https://mastodon.social/@tvler/111177846049070128
